README.md

```diff
@@ -1,2 +1,54 @@
-# tokens-distribution-spike
-If you know you know
+# Tokens Distribution Spike
+## Introduction
+
+This is the initial spike for design token build and distribution infrastructure using style-dictionary.
+Given a spec declared in the properties folder, tokens will be output via style-dictionary to specified formats.
+Custom formats are specified in the config folder under ./customFormats, these are then registered with StyleDictionary in the build.js file.
+
+## Getting Started
+To see this workflow in action, run:
+
+```
+yarn && yarn build
+```
+
+This will build the relevant output files to folders in the packages/ directory.
+Each output format is built into a publishable package such that consumers can:
+
+```
+import colors from '@ds-tokens/design-tokens-css/colors';
```

Comment:
is this accurate? I wasn't aware we had already released packages to npm. If not, maybe make it clear this is an example. Also, the package.jsons for the formats say `@dstokens` if that matters.

Reply:
yup, they're published at the moment under the dstokens namespace (will fix the example)

Reply:
faaaancy!!!